modules/govuk/manifests/apps/manuals_publisher.pp

```diff
@@ -63,6 +63,7 @@
   $redis_host = undef,
   $redis_port = undef,
   $secret_token = undef,
+  $link_checker_api_secret_token = undef,
```

Comment:
Could we add a line of documentation for this?

Reply:
Updated @thomasleese